data/service/data-trigger.js

```diff
@@ -322,46 +319,76 @@ exports.DataTrigger.prototype = Object.create(
 
                     // Start an asynchronous fetch of the property's value if necessary.
                     this.getObjectProperty(object).catch((error) => {
-                        console.error(this._propertyName + " DataTrigger: getObjectProperty error: ", error);
+                        console.error(propertyName + " DataTrigger: getObjectProperty error: ", error);
                     });
                 }
 
                 //}
 
-                /**
-                 * We're going to initialize to-Many to their collection type.
-                 * I don't think we need to check our valueStatus to see if
-                 * a fetch is in flight or have been made, since we merge local value
-                 * and fetched ones in the setter.
-                 */
+                // Ensure to-Many properties are initialized to their collection type.
+                this._ensureCollectionValue(object);
+
+                // Return the property's current value.
+                return getter ? getter.call(object) : object[this._privatePropertyName];
```

Comment:
same for getter variable, it's only used twice, so using this._valueGetter should be better